src/components/TasksTable/index.jsx

```diff
@@ -92,10 +102,23 @@ export default class TasksTable extends Component {
     });
   };
 
+  handleChipClick = tag => event => {
+    const query = this.getQuery();
+    const newQuery =
+      query.tag === tag ? omit(['tag'], query) : { ...query, tag };
```

Comment:
Why are we omitting the `tag` key? Also, this should probably be logic handled by `getQuery`.